PracticeGit/intlist.c

```diff
@@ -0,0 +1,107 @@
+//----------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+//------------------------------------------------------------------------------------------------
+//   intlist.c
+//   Program on Linked List functions.
+//--------------------------------------------------------------------------------------------
+#include <malloc.h>
+#include "intlist.h"
+
+// Initialize the head pointer of the linked list to NULL.
+struct node* head = NULL;
+
+struct node* CreateList (int initialData) {
+   struct node* head = (struct node*)malloc (sizeof (struct node));

```

Comment:
User may not want to give initial data. You should keep the two operations (creation of list and adding data) separate